src/write_gcm.c

```diff
@@ -2813,7 +2824,8 @@ static void wg_json_RFC3339Timestamp(json_ctx_t *jc, cdtime_t time_stamp);
 //   string collectd_version = 3;
 //   repeated CollectdPayload collectd_payloads = 4;
 // }
-static int wg_json_CreateCollectdTimeseriesRequest(_Bool pretty,
+static int wg_json_CreateCollectdTimeseriesRequest(
```

Comment:
I believe that already happens via wg_log_json_message in line 3894. The CreateTimeSeries server-side mechanism is different than for CreateCollectdTimeSeriesRequest, but the implementation is there already.